app/assets/javascripts/analytics/_scrollTracking.js

```diff
@@ -0,0 +1,29 @@
+(function (GOVUK) {
+  "use strict";
+
+  function ScrollTracking() {
+    this.trackedElements = [];
+  }
+
```

Comment:
you might wanna have another function called `ScrollTracking.init = function(){ };` otherwise the ScrollTracking class will be reinitialised. This is a default behaviour of the GOVUK.GDM object.